scripts/reports/bundle-size/command.js

```diff
@@ -1,29 +1,11 @@
-const { resolve } = require('path');
-const { readFileSync } = require('fs');
-const { promisify } = require('util');
-const exec = promisify(require('child_process').exec);
+const {resolve} = require('path');
```

Comment:
no need to build anymore

Reply:
Hmm, wouldn't we need to build the `master` branch to compare the PR branch against it?

Reply:
Woops, that's right. Gonna fix that.